src/utils.js

```diff
@@ -70,6 +70,33 @@ export const colorAsInt = (hexString) => {
     return parseInt(hexString.slice(1), 0x10)
 };
 
+/**
+ * Returns a <g> with three smaller 'endCap' triangles
+ * @param props
+ * @returns <g>
+ */
+export const endCaps = (panel, sideLength) => {
```

Comment:
This is awesome thanks! Is there a way to make this optional like as a prop in the React component in case people want to turn it on/off?

Reply:
Absolutely.  Let me take a pass at that and ping you when I've pushed it up.

Reply:
Naming things is hard, but I went with `removeEndCaps` since visually the end of the triangle is being removed